File: ws/camacho/U1/ws_10/JavaApplication9/src/ec/edu/espe/javastudents/model/Student.java
package ec.edu.espe.javastudents.model;

/**
 *
 * @author mateo
 */
public class Student {
    private int id;
    private String name;
    private String lastname;
    private String email;
    private int nrc; // Nuevo atributo NRC

    @Override
    public String toString() {
        return "Student{" + "id=" + id + ", name=" + name + ", lastname=" 
                + lastname + ", email=" + email + ", nrc=" + nrc + '}';
    }

    public Student(int id, String name, String lastname, String email, int nrc) {
        this.id = id;
        this.name = name;
        this.lastname = lastname;
        this.email = email;
        this.nrc = nrc;
    }

    // Getters y setters
    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getLastname() {
        return lastname;
    }

    public void setLastname(String lastname) {
        this.lastname = lastname;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public int getNrc() {
        return nrc;
    }

    public void setNrc(int nrc) {
        this.nrc = nrc;
    }
}

File: EXAM/jami/u2/q71_100_Jami_Klever/Universities/model/Student.py
class Student:
    def __init__(self, id, name, lastName):
        self._id = id
        self._name = name
        self._lastName = lastName

    # Getters
    def get_id(self):
        return self._id

    def get_name(self):
        return self._name

    def get_lastName(self):
        return self._lastName

    # Setters
    def set_id(self, id):
        self._id = id

    def set_name(self, name):
        self._name = name

    def set_lastName(self, lastName):
        self._lastName = lastName
